ServiceDesk.BLL/Services/GenericService.cs

```
@@ -57,16 +58,17 @@ public async Task<IEnumerable<TModel>> GetListByPredicateAsync(Expression<Func<T
         return _mapper.Map<TModel>(entity);
     }
 
-    public async Task UpdateModelAsync(TModel model, CancellationToken cancellationToken)
+    public async Task<TModel?> UpdateModelAsync(Guid id, TModel model, CancellationToken cancellationToken)
     {
-        var entity = await _genericRepository.GetEntityByIdAsync(model.Id, cancellationToken);
+        var entity = await _genericRepository.GetEntityByIdAsync(id, cancellationToken);
 
         if (entity == null)
         {
-            return;
+            return null;
```

It will be better to throw an exception here